core/frontend/src/render/RealityMeshParams.ts

```diff
@@ -16,6 +16,16 @@ import {
 import { GltfMeshData } from "../tile/internal";
 import { Mesh } from "./primitives/mesh/MeshPrimitives";
 
+function precondition(condition: boolean, message: string | (() => string)): asserts condition {
```

Comment:
I'd like to add this function to @itwin/core-bentley to make it easy to test for violations of API contracts in situations where the function can't proceed if the caller passed invalid arguments.
Any objection @kabentley @grigasp?

Reply:
I believe in most situations TS provides means for defining API in a way that doesn't require runtime checks like this. But I'll also admit that sometimes there are situations where that's not possible, so having this function at hand would be useful. Just maybe add a note recommending to think about API first and only using this as the last alternative.

Reply:
Let's leave it alone for now. Hopefully there's not much need for it.